### PR TITLE
Issue 371: Moved Teach LA images to our own site and used the Image tag

### PR DESCRIPTION
**Pull request by ichettiath**
The profile pics of the Teach LA officers were taken from Teach LA's website. The images were moved to our own site and `` tags were replaced with `<Image />` tags.

### CODE REVIEW

**Review by advaithg — CHANGES_REQUESTED** [review]
This looks good! Thanks for doing this + also switching to `Next/Image`
The only change I have is to reorder the `Image from next\image` import to be above the react import, like the warning says.

**Review by advaithg — COMMENTED** [review]
something like "closes #371" or "resolves #371" in the issue description will automatically close the issue as well

File: components/Impact/PersonCard.js [hunk 1]
```diff
@@ -6,7 +7,7 @@ export default function PersonCard({name, pronouns, position, major, year, img,
 		<div className={`${styles['workshop-card']} ${styles['grid-tablet-only-2']}`}>
 			<div className={styles['workshop-image-container']}>
 				{/* eslint-disable-next-line @next/next/no-img-element */}
```
Comment (mattxwang) [inline components/Impact/PersonCard.js:9]: Should remove this comment now

File: components/JEDI/JEDICard.js [hunk 2]
```diff
@@ -11,7 +12,9 @@ function JEDICard({ name, pronouns, bio, pic, alt }) {
   return (
     <div className={styles['mb-2']}>
       {/* eslint-disable-next-line @next/next/no-img-element */}
```
Comment (mattxwang) [inline components/JEDI/JEDICard.js:14]: Should also remove this comment now

**Review by mattxwang — COMMENTED** [review]
I think there's some cleanup for this PR that needs to be done, @advaithg @ichettiath

**Pull request merged**